FrontEnd/src/components/basics/Fallback/Fallback.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+import { amenityToIcon, Amenity } from '@basics/Amenities';
+import { Icon as IconType, miscIcons } from '@icons';
+
+interface FallbackProps {
+  amenity: Amenity;
+}
+
+// isAmenity is a type guard to check if key is of Amenity type
+const isAmenity = (key: any): key is Amenity => {
+  return key in amenityToIcon;
+};
+
+// Fallback Icon Component returns an amenity Icon if it is in the list of amenity icons
+// otherwise, returns a fallback Icon
+const FallbackIcon: FunctionComponent<FallbackProps> = ({ amenity }) => {
```

Comment:
Nit: change to `AmenityIcon` and move component to under `Amenities` folder since this'll mostly be used in the Amenities component (also change to `AmenityIcon.tsx` or something similar rather than `Fallback.tsx`)